.github/workflows/test.yml

```diff
@@ -15,7 +15,13 @@ jobs:
       fail-fast: false
       matrix:
         os: [macos-latest, ubuntu-latest, windows-latest]
+        vscode_version: [stable]
+        include:
+          - os: ubuntu-latest
+            vscode_version: 1.66.0
```

Comment:
Note that we now test on VSCode version `1.66.0`.  I figured two versions back was good enough.  We can bump this one in the future if we want